EmailNotifications/Email.cs

```diff
@@ -88,6 +106,12 @@ public void sendEmail()
                 // SEND IN HTML FORMAT (comment this line to send plain text).
                 message.IsBodyHtml = true ;
                 message.Body = outS;
+                //string sFile = @"C:\Users\Nthokozo Motsumi\Documents\WaterLog\WaterLog-Back-End\EmailNotifications\logo.PNG";
```

Comment:
Remove the comment line since the sFile variable will be getting its value from the current directory

Reply:
done